src/angular-fluidbox.js

```diff
@@ -15,7 +15,9 @@ angular.module('fluidbox', [])
         viewportFill: attrs.viewportFill || 0.95
       };
 
-      element.fluidbox(options);
+      $timeout(function () {
+        element.fluidbox(options);
+      });
```

Comment:
If you set the image source through a controller, then fluidbox won't work.
[jsfiddle](http://jsfiddle.net/A8Vgk/2461/)